### DISCUSSION TITLE
'RasterioSource' object has no attribute 'activate'

### DISCUSSION THREAD

**Discussion by Ramanmagar** [azavea/raster-vision#1719] [Q&A] [answered]
'RasterioSource' object has no attribute 'activate' .. this error coming in colab  

from rastervision.core.box import Box

window = Box(ymin=0, xmin=0, ymax=400, xmax=400)

with raster_source.activate():
  chip = raster_source.get_chip(window=window)

Thanks In advance..

**Comment by AdeelH** [comment, marked as answer]
You might be trying to run older Raster Vision code with a newer installed Raster Vision version.

**Reply by Ramanmagar** [reply]
Thank you Sir ..where I get latest code for latest raster version... on default colab old code coming.

**Reply by AdeelH** [reply]
What default Colab?

**Reply by Ramanmagar** [reply]
Sir I'm asking about updated code of Raster_Vision_Change_Detection_Tutorial ....because when I search updated code old code coming.. 

Thank You..

**Reply by AdeelH** [reply]
That [notebook](https://colab.research.google.com/drive/1Ex5xbAx4epzReCQDsqvZLoTpF6AtfnHr?authuser=1) depends on a specific branch of Raster Vision which it installs itself. It is not guaranteed to work with any other version of Raster Vision and there is no updated version of that notebook.

**Reply by Ramanmagar** [reply]
Ohk Sir, Thank you for valuable reply
